spec/models/bulkrax/oai_adventist_qdc_entry_spec.rb

```diff
@@ -41,7 +41,6 @@
                 <title>Testimony for the Church: Number 7</title>
                 <resource_type>Book</resource_type>
                 <date_created>1862-01-01</date_created>
-                <edition>Revised</edition>
```

Comment:
This is a little odd to see removed.

Reply:
cc @kirkkwang Would you happen to know or remember why this line was removed?

Reply:
nvm, it looks like it was part of [this](https://github.com/scientist-softserv/adventist-dl/commit/75147da978ffc155c016c096520d17a0411054f4) commit, which was merged into main feb 20th. Perhaps this branch is out of date? I'll try to update it w main

Reply:
I've reset this file to match main.